data/strings/sound.txt

```diff
@@ -594,6 +594,50 @@
     en = NULL
     nl = Neem
 
+  [route_recalculating]
+    en = Recalculating.
```

Comment:
In the languages I know these strings are a bit off. Something coming from "to recalculate" not "recalculating".

Reply:
@matheusgomesms feel free to suggest more natural translations for the languages you know. I personally think "recalculating" by itself of "rerouting" is most understandable but all suggestions are welcome.

Reply:
The problem is that the translations are in infinitive, instead of gerund (-ing in English). I can't confirm for all languages, but the ones I know are all off, so I suppose all other languages are off too.

I'll gladly suggest corrections to my languages, but just wanted to confirm that these are the strings that will be used, and will not be replaced by another regen.

Reply:
Maybe this would be better? (Although a shorter translation is preferred...)

```
    en = The route is being recalculated.
    af = Die roete word herbereken.
    ar = يتم إعادة حساب المسار.
    az = Marşrut yenidən hesablanır.
    be = Маршрут пералічваецца.
    bg = Маршрутът се преизчислява.
    ca = Es torna a calcular el recorregut.
    cs = Trasa se přepočítá.
    da = Ruten genberegnes.
    de = Die Route wird neu kalkuliert.
    el = Η διαδρομή υπολογίζεται εκ νέου.
    en-GB = The route is recalculated.
    es = Se vuelve a calcular la ruta.
    et = Marsruut arvutatakse uuesti.
    eu = Ibilbidea berriro kalkulatu da.
    fa = مسیر دوباره محاسبه می شود.
    fi = Reitti lasketaan uudelleen.
    fr = L'itinéraire est recalculé.
    he = המסלול מחושב מחדש.
    hi = मार्ग की पुनर्गणना की गई है.
    hu = Az útvonal újraszámításra kerül.
    id = Rute dihitung ulang.
    it = Il percorso viene ricalcolato.
    ja = ルートが再計算されます。
    ko = 경로가 다시 계산됩니다.
    lt = Maršrutas apskaičiuojamas iš naujo.
    mr = मार्गाची पुनर्गणना केली जाते.
    nb = Ruten beregnes på nytt.
    nl = De route wordt opnieuw berekend.
    pl = Trasa zostanie ponownie obliczona.
    pt = A rota é recalculada.
    ro = Traseul este recalculat.
    ru = Маршрут пересчитывается.
    sk = Trasa sa prepočíta.
    sv = Rutten beräknas på nytt.
    sw = Njia imehesabiwa upya.
    th = เส้นทางจะถูกคำนวณใหม่
    tr = Rota yeniden hesaplanır.
    uk = Маршрут перераховується.
    vi = Lộ trình được tính toán lại.
    zh-Hans = 重新计算路线。
    zh-Hant = 路線重新計算。
```

Reply:
Ahh I see. The problem is Deepl itself. When I use Google Translate for the previous string (en=Recalculating the route), it gives me correct translations. Deepl, for some reason, goes from gerund to infinitive, even if I append "it is" in the beginning.

What about this?

en = Recalculating the route
af = Herbereken die roete
ar = إعادة حساب الطريق
az = Marşrutun yenidən hesablanması
be = Пераразлік маршруту
bg = Преизчисляване на маршрута
ca = Recalcul de la ruta
cs = Přepočítávání trasy
da = Genberegning af ruten
de = Neuberechnung der Route
el = Επανυπολογισμός της διαδρομής
en-GB = Recalculating the route
es = Recalculando la ruta
et = Marsruudi ümberarvutamine
eu = Ibilbidea berriro kalkulatzea
fa = محاسبه مجدد مسیر
fi = Reitin uudelleenlaskenta
fr = Recalculer l'itinéraire
he = חישוב מחדש של המסלול
hi = मार्ग की पुनर्गणना
hu = Az útvonal újratervezése
id = Menghitung ulang rute
it = Ricalcolo del percorso
ja = ルートの再計算
ko = 경로 재계산 중
lt = Maršruto perskaičiavimas
mr = मार्गाची पुनर्गणना करत आहे
nb = Rekalkulerer ruten
nl = De route opnieuw berekenen
pl = Ponowne obliczanie trasy
pt = Recalculando a rota
ro = Recalcularea traseului
ru = Перерасчет маршрута
sk = Prepočítavanie trasy
sv = Beräknar om rutten
sw = Kuhesabu upya njia
th = การคำนวณเส้นทางใหม่
tr = Rotanın yeniden hesaplanması
uk = Перерахунок маршруту
vi = Tính toán lại lộ trình
zh-Hans = 重新计算路线
zh-Hant = 重新計算路線